apps/omg_status/test/omg_status/release_tasks/set_tracer_test.exs

```diff
@@ -31,6 +31,14 @@ defmodule OMG.Status.ReleaseTasks.SetTracerTest do
       # it got fiddled before
       :ok = Application.put_env(@app, Tracer, @configuration_old, persistent: true)
       :ok = Application.put_env(@app, :spandex_datadog, @configuration_old_spandex_datadog, persistent: true)
+      :ok = System.delete_env("HOSTNAME")
+      :ok = System.delete_env("DD_DISABLED")
+      :ok = System.delete_env("APP_ENV")
+      :ok = System.delete_env("DD_PORT")
+      :ok = System.delete_env("DD_HOSTNAME")
+      :ok = System.delete_env("DD_APM_PORT")
+      :ok = System.delete_env("BATCH_SIZE")
+      :ok = System.delete_env("SYNC_THRESHOLD")
```

Comment:
This seems off to me. Deleting several System env vars, unrelated to `omg_status` here. If we need to delete these env vars, it seems like each var would have a home where it would make sense to delete them.

Reply:
they're related to omg status because that's where datadog connection boots from.